CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+-
+
+# v6.3.0
+
+- [added] `admin.projectManagement()` API for managing apps in a Firebase
```

Comment:
Please move the notes from lines 27-29 about the ProjectManagement service from the v6.1.0 section to here.

Reply:
Done